pages/index.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import Home from '.';
+
+describe(Home, () => {
+  it('renders a heading', () => {
+    render(<Home />);
+
+    const heading = screen.getByRole('heading', {
+      name: /welcome to next\.js!/i,
+    });
+
+    expect(heading).toBeInTheDocument();
```

Comment:
If you agree on setting up the test id attribute on the [previous PR](https://github.com/tyrro/react-survey/pull/47#discussion_r831760337) maybe use it here too 💭 
We can also [configure the test id attribute](https://testing-library.com/docs/dom-testing-library/api-configuration/) so when we add the test id to the component we can use it on both UI and unit test 🙌🏼 

```suggestion
    const heading = screen.getByTestId(headingTestId);

    expect(heading).toBeVisible();
    expect(heading).toHaveTextContent(...);
```